app/models/coin.rb

```diff
@@ -110,12 +110,12 @@ def summary
       ]
     end
 
-    most_common_feed_source_name = self.most_common_feed_source.name
-    most_common_news_category_name = self.most_common_news_category.name
+    most_common_feed_source_name = self.most_common_feed_source&.name
+    most_common_news_category_name = self.most_common_news_category&.name
     result << %W[
       The most common news source covering #{self.name} is #{most_common_feed_source_name} and the
       most common news category is #{most_common_news_category_name}.
-    ]
+    ] if most_common_feed_source_name != nil and most_common_news_category_name != nil
```

Comment:
Thanks for the catch @anthotsang !

Jesse has an existing fix that's already deployed though, so let's roll this one back please to handle the merge conflict.

Reply:
Reverted